### ISSUE TITLE
backend shows deleted records to everyone except admin/reviewer

### ISSUE THREAD

**Issue by nics** [LibreCat/LibreCat#282]
see lib/LibreCat/App/Catalogue/Route/search.pm

**Comment by nics** [comment]
fixed in a9b78ba6e9acbe97b65f12120d059002985b8a4f

**Comment by verheyenkoen** [comment]
Added frontend test:
```
$(npm bin)/cypress run --spec cypress/integration/issues/issue-282.spec.js 
```
